client/app/dbaas/logs/detail/offer/logs-offer.html

```diff
@@ -2,18 +2,21 @@
     <oui-back-button class="back_button_inline_text" data-ui-sref="dbaas.logs.detail.home"></oui-back-button>
     <span class="oui-header_3">Change offer</span>
 
-    <h3 data-translate="logs_offer_select_offer" class="pl-4"></h3>
-    
+    <div class="pl-4">
+        <h3 ng-if="ctrl.offerDetail.currentOffer !== 'logs-pro-0001'" data-translate="logs_offer_description_pro" translate-value-offer="{{ctrl.offerDetail.currentOffer | translate}}"></h3>
+        <h3 ng-if="ctrl.offerDetail.currentOffer === 'logs-pro-0001'" data-translate="logs_offer_description_basic"></h3>
```

Comment:
why not use a "pro" property in each offer?  If currentOffer.pro => logs_offer_description_pro else : logs_offer_description_basic

Reply:
Ahhh, there is no property to determine if its a basic offer or log offer, the currentOffer object has only one reference which is offer code like 'logs-pro-001' or 'logs-pro-015'. Only 'logs-pro-001' is called basic/discovery offer. I can manually inject a property but i have to iterate through the entire list to do that! 
Should i do that ?

Reply:
Yeah I would do that... I think it'd be clear that way